### PR TITLE
Walkthru development

### PR DESCRIPTION
**Pull request by Leo2452**
After merging all branches, only added one commit (last line) from walkthrough. Branch created to make pull requests as I walk through walkthrus.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -26,3 +26,88 @@ function addRandomGreeting() {
...
+    fetch('/data').then(response => response.json()).then(greeting => {
+        var box = document.getElementById('special-greeting-container');
+        box.innerText = greeting;
+    })
+}
+
+/**
+ * Adds a random fact to the page
+ */
+function addRandomFact() {
```
Comment (landondare) [inline portfolio/src/main/webapp/script.js]: A lot of these changes seem to be duplicate changes from previous pull requests. Try to submit your pending pull requests first, then merge the changes back to this branch. That way the changes won't show up twice.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -14,19 +14,41 @@
...
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that chooses a random greeting from provided list. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+	private List<String> comments = new ArrayList<>();
```
Comment (landondare) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:36]: Fix indent

File: walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html [hunk 3]
```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```
Comment (landondare) [inline walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html:13]: I'm not seeing where this function is defined.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (Leo2452) [reply portfolio/src/main/webapp/script.js]: I've just merged master into this branch and pushed the changed, but the changes persist above. Should I create a new branch to single-out the intended commits after the repeated commits? Or is there some better way to get rid of them?

File: walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html (hunk 3 above)
Comment (Leo2452) [reply walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html:13]: This walkthrough example implemented these two functions in the same random-quotes directory. There weren't any changes to the file so it wasn't added/committed, but it was part of the walkthrough

Comment (landondare) [reply walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html:13]: I see, my mistake.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (landondare) [reply portfolio/src/main/webapp/script.js]: Looks like that took care of most of the differences.

**Pull request closed without merging**